Apps/Sandcastle/gallery/Custom Post Process.html

```diff
@@ -44,23 +44,23 @@
         });
 
         const fragmentShaderSource = `
-          uniform sampler2D colorTexture;
```

Comment:
We updated some options in [the prettier merge instructions](https://github.com/CesiumGS/cesium/pull/12206#issuecomment-2379955367) to prevent unwanted trailing spaces like this from showing up.

Reply:
OK. To clean up this particular PR for now, I would recommend checking out the previous unchanged files (`Apps/Sandcastle/*`, `packages/engine/Source/Scene/ArcGisMapServerImageryProvider.js`, etc) from `main` and committing to this branch. 

CC @jjspace

Reply:
I think that it didn't work when pulling them from 'main', but with
`git checkout post-prettier-v3 -- "Apps/Sandcastle/gallery/Custom Post Process.html"`
(for all relevant files), it _seemed_ to work - at least, the files do no longer show up as 'modified' here.